grafana/provisioning/dashboards/dev/anchorservice.json

```diff
@@ -3508,6 +3508,750 @@
+            "targets": [
+                {
+                    "alias": "",
+                    "dimensions": {
+                        "DBInstanceIdentifier": "ceramic-$env-cas"
+                    },
+                    "expression": "",
+                    "id": "",
+                    "matchExact": true,
+                    "metricName": "CPUCreditUsage",
+                    "namespace": "AWS/RDS",
+                    "period": "",
+                    "refId": "A",
+                    "region": "default",
+                    "statistics": [
+                        "Sum"
+                    ]
+                }
+            ],
+            "thresholds": [],
+            "timeFrom": null,
+            "timeRegions": [],
+            "timeShift": null,
+            "title": "Postgres CPUCreditUsage (ceramic-$env-cas)",
+            "tooltip": {
+                "shared": true,
+                "sort": 0,
+                "value_type": "individual"
+            },
+            "type": "graph",
+            "xaxis": {
+                "buckets": null,
+                "mode": "time",
+                "name": null,
+                "show": true,
+                "values": []
+            },
+            "yaxes": [
+                {
+                    "$$hashKey": "object:176",
+                    "format": "short",
+                    "label": null,
+                    "logBase": 1,
+                    "max": null,
+                    "min": "0",
+                    "show": true
+                },
+                {
+                    "$$hashKey": "object:177",
+                    "format": "short",
+                    "label": null,
+                    "logBase": 1,
+                    "max": null,
+                    "min": null,
+                    "show": true
+                }
+            ],
+            "yaxis": {
+                "align": false,
+                "alignLevel": null
+            }
+        },
+        {
+            "alert": {
+                "alertRuleTags": {},
+                "conditions": [
+                    {
+                        "evaluator": {
+                            "params": [
+                                1
+                            ],
+                            "type": "lt"
+                        },
+                        "operator": {
+                            "type": "and"
+                        },
+                        "query": {
+                            "params": [
+                                "A",
+                                "5m",
+                                "now"
+                            ]
+                        },
+                        "reducer": {
+                            "params": [],
+                            "type": "avg"
+                        },
+                        "type": "query"
+                    }
+                ],
+                "executionErrorState": "alerting",
```

Comment:
should we distinguish betwen prod and dev alerts?  I'm just afraid of having too many alerts and they get muted or ignored.  But this is def important, approving

Reply:
it will go to a different discord channel so taht is fine, just for future maybe only propagate prod ones to pager duty.  so this doesn't actually impact this pr, just a note for when we do that

Reply:
well so far `/dashboards/<env>` are configured to send alerts only to `alerts-<env>` channel in discord (with the exception of dev alerts which go to a channel called `dev`).

`prod` -> `alerts-prod`
`tnet` -> `alerts-tnet`
`qa` -> `alerts-qa`
`dev` -> `dev`

So we should never see dev alerts in the prod channel, and so on

Reply:
oh sorry didn't see your second comment. yeah. agree